components/Sataserver/include/plat/pc99/supermicro/plat/sata.h

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
I think this commit should be in a separated pull request.

Reply:
It was, it's just polluted my branch, oops.